core/lib/Thelia/Config/I18n/en_US.php

```diff
@@ -47,7 +47,7 @@
     'All Products' => 'All Products',
     'All countries' => 'All countries',
     'All shipping methods' => 'All shipping methods',
-    'An error happend while copying %prev to %dest' => 'An error happend while copying %prev to %dest',
+    'An error happend while copying %prev to %dest' => 'An error happened while copying %prev to %dest',
```

Comment:
I don't agree, the key can't change. It doesn't matter if there is a typo, you can fixed it in the translation.